### ISSUE TITLE
readme outdated for drupal 8

### ISSUE THREAD

**Issue by shortthefomo** [webflo/drupal-swiftmailer#17]
the readme file is from drupal 7 and has not been updated sufficiently for drupal 8 usage

**Comment by webflo** [comment]
Right, we should add some examples. Here is an example in the meantime. https://gist.github.com/webflo/d88e2c83689028f88634
